src/reducers/Material.js

```diff
@@ -38,6 +38,7 @@ function materialsCloneOne(state, action) {
 function materialsToggleIsNonPeriodicForOne(state, action) {
     const newMaterial = state.materials[state.index].clone();
     newMaterial.isNonPeriodic = !newMaterial.isNonPeriodic;
+    newMaterial.hash = '';
```

Comment:
Let's pass as a parameter to the clone function: `material.clone({resetHash: true})`?